### ISSUE TITLE
[Designer]Bug 33489617: [FoodOrder.Json]: "Your registration is almost complete" is not defined as a heading on 'FoodOrder.Json' card.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7624]
### Problem Description

[Bug 33489617](https://microsoft.visualstudio.com/OS/_workitems/edit/33489617): [FoodOrder.Json]: "Your registration is almost complete" is not defined as a heading on 'FoodOrder.Json' card.

Test Environment:

Application: Adaptive Cards
OS version: Dev (OS Build 21387.1)

Application Version: 2.7.2105.11001

Screen Reader: Narrator

Repro steps:

1. Launch the Adaptive Card application and narrator.

2. Adaptive Card UWP Visualizer screen starts appearing. 

3. Navigate to FoodOrder.Json button by using Tab key and activate it.

4. FoodOrder adaptive card will be appeared on right sight of the pane.

5. Navigate to FoodOrder card by using Tab key.

6. Navigate in scan mode to "Your registration is almost complete" and observe the issue.

Actual Result:
"Your registration is almost complete" is not defined as a heading on 'FoodOrder.Json' card. Screen reader is only announcing as, "Your registration is almost complete".

Expected Result:
"Your registration is almost complete" should be defined under a heading. Screen reader should announce as, "Your registration is almost complete heading level 1".

User Experience:
It will be confusing & difficult for the users if Heading will not be defined inside the adaptive card.

Observation:
NVDA(2020.4) and JAWS(2021) scan mode is not working on Adaptive Cards.

Note: Same issue is also repro with "Your registration is almost complete" under "Food order with validation" card.

### Card JSON

```JSON
Designer Bug. Repro on UWP Adaptive Card visualizer
```

**Comment by vagpt** [comment]
Closing this issue as there is no need to define the text in heading.
